pipelines/placesapi/__init__.py

```diff
@@ -0,0 +1,73 @@
+"""
+ Loads location information (Ratings, Reviews, Business Hours) from Google Maps. 
+"""
+
+import dlt
+from dlt.sources.helpers import requests
+from dlt.extract.source import DltResource
+
+
+@dlt.source
+def places_api_source(api_secret_key=dlt.secrets.value) ->  DltResource:
+    """
+    The main source for dlt pipeline returns the text_search_resource.
+
+    @:params: api_secret_key - The api key for the Places API defined in dlt.secrets.
+    """
+    return text_search(api_secret_key)
+
+
+@dlt.resource(write_disposition="append")
+def text_search(api_secret_key=dlt.secrets.value):
+    """
+    Creates a text search resource. A Text Search returns information about a set of places based on a text string provided.
+
+    For more Info: "https://developers.google.com/maps/documentation/places/web-service/search-text"
+
+    The Parameters to be passed to endpoint:
+    @:params: query - The text against which a search will be made.
+    @:params: radius (optional)  - Defines the distance (in meters) within which to return place results. You may bias results to a specified circle by passing a location and a radius parameter. Default value is 50,000 meters for text search.
+    @:params: location (optional) - Can be used with the radius parameter to bias result.
+    @:params: api_secret_key - The api key for the Places API defined in dlt.secrets.
+    """
+
+    # API Endpoint.
+    text_search_url = "https://maps.googleapis.com/maps/api/place/textsearch/json?"
+
+    payload = {
+        'query' : 'ADD SEARCH TEXT HERE',
```

Comment:
We can improve the usability of this resource if we let the user of the pipeline pass these parameters via the `text_search()` arguments. Please take a look at how resources are implemented in the Chess pipeline: https://github.com/dlt-hub/pipelines/blob/beeed5001656fd42c7da7f0b955e6fd8be43ebae/pipelines/chess/__init__.py#L49-L50